spec/features/navigation/nav_bar_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "Navigation Bar" do
+  it "has the correct content" do
+    visit root_path
+
+    within "nav" do
+      expect(page).to have_link("Home", href: root_path)
+      expect(page).to have_link("Dashboard", href: user_dashboard_path)
+      expect(page).to have_link("Find Books", href: books_path)
+      expect(page).to have_link("Friends", href: user_friends_path)
+      expect(page).to have_link("My Books", href: user_books_path)
+      expect(page).to have_link("Account Info", href: user_account_path)
+    end
+  end
+  it "has working paths for all links" do
+    skip
```

Comment:
On line 17

You can pass an argument to `skip` to change the message in the terminal that says `no reason given`.

```
skip "Routes don't exist yet"
```

Reply:
That's cool. I added that in now